modules/tiles/src/tileset/tile-3d.js

```diff
@@ -435,7 +435,7 @@ export default class TileHeader {
 
   // TODO - remove anything not related to basic visibility detection
   _initializeRenderingState(header) {
-    this.depth = header.level;
+    this.depth = this.parent && this.parent.hasOwnProperty('depth') ? this.parent.depth + 1 : 0;
```

Comment:
No need to use `hasOwnProperty` unless you are messing with `Object.prototype`.
```suggestion
    this.depth = this.parent && this.parent.depth ? this.parent.depth + 1 : 0;
```